tests/sav/test_astprinter_args1.res

```diff
@@ -0,0 +1,80 @@
+# syntax_expr$Sys$main
+var a
+var b: Bool
+var c: Collection[Int] = [1]
+var d = 1
+a = 1.+(2).-(3.unary -./(4).*(5).%(6.**(2)))
+b = true and not false or true or a.<(0) or a.<=(0) or a.==(0) or a.!=(0) or a.>=(0) or a.>(0)
+a = 1.&(2).|(3.unary ~.^(4.<<(5).>>(6)))
+a = 1.<=>(2)
+c = [1..2[
+c = [1..2]
+c = [123]
+a += 1
+a -= 1
+a *= 1
+a /= 1
+a %= 1
+a **= 1
+a &= 1
+a |= 1
+a ^= 1
+a <<= 1
+a >>= 1
+a = 1
+print(a)
+print(a)
+a.abs
+a.max(2)
+a.max(2)
+a.setbit(0, 1)
+c = new Array[Int].defaultinit
+c = new Array[Int].with_capacity(10)
+a = new Array[Int].with_capacity(10).length
+c.add(1)
+a = a isa Array[Int]
+a = null
+if b then a = 1
+d = a.as(not null)
+d = a.as(Int)
+do end
+do a = 1
+do
+	a = 1
+	a = 1
+end
+if b then else end
```

Comment:
There's a pending else from probably AIfexprExpr always printing the else.

Reply:
for some reason, the else is in the ast but is empty. So I just do not print empty else parts